src/v2/Components/Authentication/Desktop/SignUpForm.tsx

```diff
@@ -164,3 +211,48 @@ export class SignUpForm extends Component<FormProps, SignUpFormState> {
     )
   }
 }
+
+const SignUpFormFragmentContainer = createFragmentContainer(SignUpForm, {
+  requestLocation: graphql`
+    fragment SignUpForm_requestLocation on RequestLocation {
+      countryCode
+    }
+  `,
+})
+
+export const SignUpFormQueryRenderer: React.FC<FormProps> = passedProps => {
+  const { relayEnvironment } = useSystemContext()
+  const variables = { ip: sd.IP_ADDRESS }
+
+  return (
+    <QueryRenderer<SignUpFormLocationQuery>
+      //  @ts-expect-error STRICT_NULL_CHECK
+      environment={relayEnvironment}
+      variables={variables}
+      query={graphql`
+        query SignUpFormLocationQuery($ip: String!) {
+          requestLocation(ip: $ip) {
+            ...SignUpForm_requestLocation
+          }
+        }
+      `}
+      render={({ error, props }) => {
+        // what's the cooler way to do this??
```

Comment:
😎

Reply:
What are you saying the code that's there is _already_ cool?? 😝